inline_verifier.go

```diff
@@ -333,7 +333,17 @@ func (v *InlineVerifier) CheckFingerprintInline(tx *sql.Tx, targetSchema, target
 		}
 	}
 
-	return v.compareHashesAndData(sourceFingerprints, targetFingerprints, sourceDecompressedData, targetDecompressedData), nil
+	mismatches := v.compareHashesAndData(sourceFingerprints, targetFingerprints, sourceDecompressedData, targetDecompressedData)
+
+	for _, mismatchedPk := range mismatches {
+		v.reverifyStore.Add(table, mismatchedPk)
+	}
```

Comment:
Should we check if `EnforceInlineVerification` is true and not add the mismatches to the`reverifyStore` in that case?